modules/tinymce/src/core/test/ts/browser/UndoManagerTest.ts

```diff
@@ -440,6 +440,35 @@ UnitTest.asynctest('browser.tinymce.core.UndoManager', function (success, failur
     ok(editor.isDirty(), 'Dirty state should be true');
   });
 
+  suite.asyncTest('Dirty state on second AddUndo', (editor, done, die) => {
+    editor.setContent('<p>a</p>');
+    LegacyUnit.setSelection(editor, 'p', 1);
```

Comment:
I was gonna suggest moving the file away from LegacyUnit... then I saw how long it was 😨

Reply:
Yeah. :/ It's a good holding pattern task  though. I'd like to go back and do it.